### PR TITLE
Move name operation into "Type" column

### PR DESCRIPTION
**Pull request by JeremyRand**
Removes redundant UI info, improving UX.

### CODE REVIEW

**Comment by JeremyRand** [comment]
Tested, works fine.

**Comment by JeremyRand** [comment]
@domob1812 This is ready for review.

File: src/qt/transactionrecord.h [hunk 1]
```diff
@@ -84,25 +84,34 @@ class TransactionRecord
         NameOp,
     };
 
+    enum NameOpType
```
Comment (domob1812) [inline src/qt/transactionrecord.h:87]: Unless there is a Qt-specific reason against it, I think you should use `enum class` here.  This makes `NameOpType` more type-safe.

**Review by domob1812 — CHANGES_REQUESTED** [review]
One small suggestion, otherwise it looks very good.  Thanks for all your UI work!

File: src/qt/transactionrecord.h (hunk 1 above)
Comment (JeremyRand) [reply src/qt/transactionrecord.h]: Good point.  I was originally trying to copy the surrounding style, but you're correct that the Bitcoin Core policy is that new code should use scoped enums.  Fixed now.

**Comment by domob1812** [comment]
Thanks for the updates, ACK 279743d6ddf554eaeaea93019477a22605642c10.

**Pull request merged**